homeworks/yaroslav_sakhno/hw01/test.py

```diff
@@ -0,0 +1,43 @@
+import sys
+import argparse
+from pyats.aetest import Testcase, test, main
+from calculation import add, divide
+
+
+class TryTest(Testcase):
+
+    parameters = {
```

Comment:
@jsakhno Please set default arguments in the args parser and remove this dictionary.

Reply:
@extsoft fixed